x-pack/test/observability_ai_assistant_api_integration/common/create_llm_proxy.ts

```diff
@@ -278,7 +282,7 @@ export class LlmProxy {
               requestBody,
               status: once((status: number) => {
                 response.writeHead(status, {
-                  'Elastic-Interceptor': name,
+                  'Elastic-Interceptor': name.replace(/[^a-zA-Z0-9 ]/g, ''), // Keeps only alphanumeric characters and spaces
```

Comment:
nit:
Do we want `titleconversation` here or `title conversation`.
If the latter:
```suggestion
                  'Elastic-Interceptor': name.replace(/[^a-zA-Z0-9 ]/g, ' '), // Keeps only alphanumeric characters and spaces
```